package.json

```diff
@@ -39,6 +39,7 @@
     "react-native-console-time-polyfill": "^1.2.1",
     "react-native-device-info": "1.6.1",
     "react-native-dialog": "^5.6.0",
+    "react-native-document-picker": "Elyx0/react-native-document-picker#v3",
```

Comment:
We don't want this experimental branch landing master: https://github.com/Elyx0/react-native-document-picker/tree/v3#warning-experimental-branch-of-v3-

Reply:
I have some complications with the original , do you really want to use original one ???